mhm/__init__.py

```diff
@@ -0,0 +1,52 @@
+import os
+import json
+import pandas as pd
+
+
+PARAMS_INDIVIDUAL = 'params_individual.json'
+PARAMS_MODEL = 'params_model.json'
+
+
+def read_json_as_dict(fpath: str):
+    """Read content of a JSON file as Python dictionary.
+
+    Args:
+        fpath (str): file path.
+
+    Returns:
+        dict: result dictionary.
+    """
+    assert os.path.isfile(fpath), 'File not found: %s.' % fpath
+    with open(fpath) as file:
+        data = json.load(file)
+    return data
+
+def create_empty_hypotheses(dir_params: str):
+    """Create empty hypotheses files in a user-defined folder.
+
+    Args:
+        dir_params (str): folder that contains the agent and model parameter files.
```

Comment:
See whether you think this makes it clearer

```suggestion
        dir_params (str): The directory of the folder that contains the agent and model parameter files.
```